subsys/bluetooth/mesh/main.c

```diff
@@ -525,7 +525,6 @@ int bt_mesh_start(void)
 
 		if (IS_ENABLED(CONFIG_BT_MESH_GATT_PROXY)) {
 			(void)bt_mesh_proxy_gatt_enable();
-			bt_mesh_adv_gatt_update();
```

Comment:
Just currious: Why does this need to be removed?

Reply:
`bt_mesh_adv_gatt_update` is needed reschedule advertiser when we change gatt advertising state in mesh (when we need to start or stop advertising (Private) Node Identity or Network ID). Before this change, the service registration was happening in place and so to start advertising the Mesh Proxy Service we needed to call `bt_mesh_adv_gatt_update` right after `bt_mesh_proxy_gatt_enable`. Now, since the service is registered in `svc_reg_work_handler`, there is no need to reschedule the advertiser here and this call is moved to `svc_reg_work_handler`.